agents/craftassist/craftassist_agent.py

```diff
@@ -133,6 +133,7 @@ def init_memory(self):
         """Intialize the agent memory and logging."""
         self.memory = mc_memory.MCAgentMemory(
             db_file=os.environ.get("DB_FILE", ":memory:"),
+            coordinate_transforms=self.coordinate_transforms,
```

Comment:
need to review this commit @kavyasrinet @soumith

Reply:
Why are we moving `coordinate_transforms` to memory ?

Reply:
its needed to run LinearExtentAttributes, which are defined in the interpreter folder (not the craftassist or robot or whatever particular interpreter folder.   So either we subclass LinearExtentAttribute for each agent, or we store the coordinate transforms somewhere where the Attribute can get them.  note that other parts of agent might also use the coordinate transforms...   

what would you rather?